ModManager_Classes/Utils/ModLoaderInstaller.cs

```diff
@@ -39,11 +39,12 @@ public async Task InstallAsync()
             }
 
             var modloaderRepo = new GithubRepoInfo() { Name = "anno1800-mod-loader", Owner = "xforce" };
-            string downloadResult = await GithubDownloader!.DownloadReleaseAsync(modloaderRepo, "loader.zip");
+            var downloadResult = await GithubDownloader!.DownloadReleaseAsync(modloaderRepo, "loader.zip", progress);
```

Comment:
GithubDownload is created in the constructor. It's better to make it non-nullable than using `!`.

Reply:
nice catch, resolved in [c77d9a0](https://github.com/anno-mods/iModYourAnno/pull/69/commits/773d9a07832adab927f33fb0382f5fd505624a7e)